coding/localizable_string.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "coding/reader.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include <cstdint>
+
+using LocalizableString = std::unordered_map<uint8_t, std::string>;
+using LocalizableStringSubIndex = std::map<uint8_t, uint32_t>;
```

Comment:
It would be nice to add a comment describing semantics of the key and value.

Reply:
`#include <map>`